src/ds_caselaw_utils/test_courts.py

```diff
@@ -67,6 +67,20 @@ def test_loads_court_by_param(self):
         repo = CourtsRepository(data)
         self.assertEqual("Court 2", repo.get_by_param("court2").name)
 
+    def test_raises_on_unknwon_court_param(self):
+        data = [
+            {
+                "name": "court_group1",
+                "courts": [{"code": "court1", "name": "Court 1"}],
```

Comment:
These courts should have `param`s not `codes`. The test's behaviour is correct (a court with the `param` of `court3` doesn't exist), but this feels a bit off.